.github/scripts/wipe-azure.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
```

Comment:
```suggestion
#!/usr/bin/env bash
```

`/bin/bash` is not portable, various systems have it in different locations. Only `/bin/sh` is guaranteed to be in `/bin` as required by POSIX, and `env` being in `/usr/bin` is a de-facto standard.

For example, on my Linux system bash is `/run/current-system/sw/bin/bash` and `/bin/bash` doesn't exist.

Reply:
(non blocking here because we know `/bin/bash` exists on CI, but generally important in other cases)